src/figuresClasses.ts

```diff
@@ -1,19 +1,90 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
 
+function round(area: number): number {
+  return Math.trunc(area * 100) / 100;
 }
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
```

Comment:
and then you can use it like this
```suggestion
  shape = Shape.TRIANGLE;
```